Formula/dar.rb

```diff
@@ -27,11 +40,18 @@ class Dar < Formula
   depends_on "upx" => [:build, :optional]
 
   def install
+    if build.stable? && ENV.compiler == :clang && MacOS.version >= :mavericks
+      ENV.libstdcxx
+    elsif build.devel?
+      ENV.O0
+      ENV.deparallelize
```

Comment:
I'd also like the facts on Xcode 8's clang first (does it work? does it also need the ENV.O0?), but yeah this is quite gross and definitely merits reporting.

Also, the deparallelize may be appropriate for stable too. I haven't tried to break it.